requirements.txt

```diff
@@ -0,0 +1,2 @@
+eth-abi>=2.1.1
```

Comment:
why do we need `requirements.txt` if `pyproject.toml` is already here?

Reply:
Removed